tests/test_psfCandidate.py

```diff
@@ -82,7 +82,7 @@ def createFakeSource(x, y, catalog, exposure, threshold=0.1):
     source['centroid_x'] = x
     source['centroid_y'] = y
 
-    exposure.getMaskedImage().getImage()[x, y] = 1.0
+    exposure.image[x, y, afwImage.PARENT] = 1.0
```

Comment:
Shouldn't this be `LOCAL` to replicate the old behavior?

Reply:
I *think* this was one of the places where I looked carefully and then determined that they're equivalent.  But in retrospect it's probably better to just use `LOCAL` and not rely to much on the carefulness of that looking.